### ISSUE TITLE
Game crashes upon startup, even with latest version of AsmodeusCore

### ISSUE THREAD

**Issue by scientificindividual** [BlesseNtumble/GalaxySpace#625]
**Do not remove this template, without it your issue will not be considered!**

**Please fill in the form below:**
------------------------------------------------------------------------
1. Minecraft version: 1.12.2
2. Galacticraft version: 4.0.2.280
3. GalaxySpace version: 2.0.19
4. AsmodeusCore version (for 2.0.1 version and above): 0.0.31
5. Side (Single player (SSP), Multiplayer (SMP), or SSP opened to LAN (LAN)): SSP
6. Other add-ons: BlankPlanet, Interstellar Exoplanets, More Planets, More Planets Extras, Zollern Galaxy, Tinker's Construct
------------------------------------------------------------------------
Description of the issue:
The game crashes upon startup, prompting the following message:

The game crashed whilst there was a severe problem during mod loading that has caused the game to fail
Error: net.minecraftforge.fml.common.LoaderExceptionModCrash: Caught exception from GalaxySpace (galaxyspace)

Exit code is -1. I looked through the issues reported on this page to see if anyone had experienced a similar issue, and I found #600, but I am already running the latest version of AsmodeusCore.
------------------------------------------------------------------------
Screenshot/Video:
<img width="953" alt="crash" src="https://user-images.githubusercontent.com/112009302/186483586-b58f80fa-4c3e-4d96-af17-e1105d9d03b3.png">

------------------------------------------------------------------------
Attached log file (or url on pastebin.com):
https://pastebin.com/D2S9uzA1

------------------------------------------------------------------------

**Comment by kaguyahimeinthemoon** [comment]
Update Galacticraft to 4.0.3:
http://curseforge.com/minecraft/mc-mods/galacticraft-legacy/

**Comment by ANerdyNub** [comment]
> Update Galacticraft to 4.0.3: http://curseforge.com/minecraft/mc-mods/galacticraft-legacy/

Wait I'm so confused I have same problem, but shouldn't the latest galacticraft version be from their own site?

**Comment by kaguyahimeinthemoon** [comment]
> > Update Galacticraft to 4.0.3: http://curseforge.com/minecraft/mc-mods/galacticraft-legacy/
> 
> Wait I'm so confused I have same problem, but shouldn't the latest galacticraft version be from their own site?

They moved to Curseforge:
https://github.com/micdoodle8/Galacticraft

**Comment by ANerdyNub** [comment]
> > > Update Galacticraft to 4.0.3: http://curseforge.com/minecraft/mc-mods/galacticraft-legacy/
> > 
> > 
> > Wait I'm so confused I have same problem, but shouldn't the latest galacticraft version be from their own site?
> 
> They moved to Curseforge: https://github.com/micdoodle8/Galacticraft

yeah I saw but they should still say something on the official site that has been used for a long while

**Comment by MrRedstoneA1** [comment]
Thanks for the info! I didn't know galacticraft had moved to one mod on curseforge. I would add some tips here for anyone else having trouble:
Don't forget to delete the old galacticraft and its dependencies.
Delete MicdoodleCore, it is no longer needed.
Make sure to edit the config files for compatibility. Follow this guide to not crash:
https://docs.mjrlegends.com/ExtraPlanets/1.12.2compact/
